From 603b7ea9c133019c9cdbd87f27bd60bce937890f Mon Sep 17 00:00:00 2001
From: Rex Lu <rex.lu@mediatek.com>
Date: Thu, 8 Jan 2026 11:17:27 +0800
Subject: [PATCH 7/7] mtk: mt76: mt7996: adjust uni cmd timeout value for
 special cmd

align logan to adjust MCU_UNI_CMD_EFUSE_CTRL and MCU_UNI_CMD_EXT_EEPROM_CTRL cmd timeout value,
otherwise console may false alarm to show mcu cmd timeout error

Signed-off-by: Rex Lu <rex.lu@mediatek.com>
---
 mt7996/mcu.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mt7996/mcu.c b/mt7996/mcu.c
index aaf4309e5..9c99cc2d7 100644
--- a/mt7996/mcu.c
+++ b/mt7996/mcu.c
@@ -275,7 +275,8 @@ mt7996_mcu_set_timeout(struct mt76_dev *mdev, int cmd)
 		mdev->mcu.timeout = 2 * HZ;
 		return;
 	case MCU_UNI_CMD_EFUSE_CTRL:
-		mdev->mcu.timeout = 20 * HZ;
+	case MCU_UNI_CMD_EXT_EEPROM_CTRL:
+		mdev->mcu.timeout = 30 * HZ;
 		return;
 	default:
 		break;
-- 
2.45.2

